app/bundles/ApiBundle/Tests/Functional/Oauth2Test.php

```diff
@@ -79,7 +79,7 @@ public function testAuthWithInvalidAccessToken(): void
 
     public function testAuthWorkflow(): void
     {
-        $this->client->enableReboot();
+        $this->client->disableReboot();
```

Comment:
I adopted this solution because this test failed, This is the [source](https://stackoverflow.com/questions/62328408/cannot-set-session-id-after-the-session-has-started-at-mockarraysessionstorage-p)

Reply:
It's disabled by default in `MauticMysqlTestCase` so this could be removed, but it's not necessary.